### DISCUSSION TITLE
Does ignorePaths override defaults?

### DISCUSSION THREAD

**Discussion by ttrieu9** [renovatebot/renovate#13249] [Request Help] [answered]
Hello! I'm was looking at the documentation for the `ignorePaths` option in the [renovate docs](https://docs.renovatebot.com/configuration-options/#ignorepaths). What was unclear to me: if I specify a value(s) for `ignorePaths` does it override the defaults (e.g. `**/node_modules/**`) or does it add on to the list of default paths to ignore?

**Comment by viceice** [comment, marked as answer]
You are overwriting that value

**Reply by ttrieu9** [reply]
Thanks!
